pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -2147,46 +2121,46 @@
             </item>
            </widget>
           </item>
-          <item row="3" column="0" colspan="2">
+          <item row="10" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout_9">
-            <item row="0" column="1">
-             <widget class="QCheckBox" name="skipPresentingDuplicateFrames">
+            <item row="2" column="0">
+             <widget class="QCheckBox" name="disableMailboxPresentation">
               <property name="text">
-               <string>Skip Presenting Duplicate Frames</string>
+               <string>Disable Mailbox Presentation</string>
```

Comment:
Could we use this PR to also add a comment for translators? Here's an example:
```
<string extracomment="Mailbox Presentation: a type of graphics-rendering technique that has not been exposted to the public that often, so chances are you will need to keep the word \"mailbox\" in English. It does not have anything to do with postal mailboxes or email inboxes/outboxes.">Disable Mailbox Presentation</string>
```

Reply:
Hopefully this is correct if you can confirm.

Reply:
Make sure the quotation marks of "mailbox" are properly escaped.

Besides that, LGTM, though if anyone can properly summarize how does this feature work, we translators would appreciate it.